lib/ex_cycle/string_builder.ex

```diff
@@ -0,0 +1,102 @@
+defmodule ExCycle.StringBuilder do
+  @moduledoc """
+  This module is in charge to build a stringify version of a `ExCycle.Rule`.
+
+  ## Examples
+
+
+      iex> ExCycle.Rule.new(:weekly, days: [:monday], hours: [10], minutes: [30])
+      ...> StringBuilder.traverse_validations(&stringify/2)
+      "daily at 10:00, 10:30"
+
+  > NOTE: You have to implement the stringify function by your side.
```

Comment:
Maybe move this before the examples
Also:

```suggestion
  > NOTE: You **MUST** implement the stringify function on your side.
```